spec/rspec/matchers_spec.rb

```diff
@@ -24,6 +24,16 @@
       end
     end
   end
+
+  describe "#respond_to?" do
+    it "returns true for methods in example scope" do
```

Comment:
This doc string isn't very communicative, IMO.  I think something like this would read better:

``` ruby
it "handles dynamic matcher methods"
```